youtube_dl/extractor/nuevo.py

```diff
@@ -0,0 +1,33 @@
+# encoding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+
+from ..utils import float_or_none
+
+class NuevoBaseIE(InfoExtractor):
+    def _extract_nuevo(self, config_url, video_id, info=None, ignore_hd=False):
```

Comment:
Also there is no need in `info` here - you can return info dict from here and add more fields at the call place once it's returned.